src/components/paragraphs/sum-testimonial-banner/sum-testimonial-banner-paragraph.tsx

```diff
@@ -156,7 +156,7 @@ const SumTestimonialBannerParagraph = ({paragraph, ...props}: Props) => {
 
           {paragraph.sumTestimonialButton?.url && (
             <ActionLink
-              className="btn rs-mt-5 w-fit rounded-full border-2 border-transparent bg-digital-red px-8 py-4 font-normal text-white transition hocus:border-white hocus:text-white"
+              className="btn rs-mt-5 flex w-fit items-center rounded-full border-2 border-transparent bg-digital-red px-8 py-4 font-normal text-white transition hocus:border-white hocus:text-white"
```

Comment:
The action link already has these classes. this is redundant.
https://github.com/SU-SWS/summer-nextjs/blob/1.x/src/components/elements/action-
link.tsx#L18